emulator_ssl.py

```diff
@@ -366,6 +369,9 @@ def main():
         argparser.add_argument('-port', type=int, default=5000,
                            help='Port to run the emulator on. Default is 5000')
 
+    MOCKUPFOLDERS = config['MOCKUPFOLDERS']
```

Comment:
We should add try statements around this (like in the non SSL version of the tool) in case the user does not supply the configuration option.